### PR TITLE
Add HUNT v2 ZAP Add-on

### PR DESCRIPTION
**Pull request by cak**
### Build the add-on

```sh
gradle build
```

Add-on ZAP file will be located at: `./build/zapAddOn/bin`

### Load the add-on

1. Open OWASP ZAP
2. File
3. Load Add-on file
4. Select HUNT `.zap` file

### CODE REVIEW

File: Remix/ZAPRemix/src/main/kotlin/ExtensionHunt.kt [hunk 1]
```diff
@@ -0,0 +1,43 @@
+package org.zaproxy.zap.extension.hunt
+
+import org.parosproxy.paros.extension.ExtensionAdaptor
+import org.parosproxy.paros.extension.ExtensionHook
+import org.parosproxy.paros.network.HttpSender
+import java.awt.EventQueue
+
+class ExtensionHunt : ExtensionAdaptor(NAME) {
+    private val huntPanel = HuntPanel()
```
Comment (thc202) [inline Remix/ZAPRemix/src/main/kotlin/ExtensionHunt.kt:9]: The creation of the UI panel should be delayed until it's know that there's a view (e.g. `initView(ViewDelegate)` or in the hook check) to work properly in headless/daemon.

Comment (cak) [reply Remix/ZAPRemix/src/main/kotlin/ExtensionHunt.kt:9]: Thanks for the review @thc202! I'll get that fixed ASAP!

**Comment by cak** [comment]
@thc202 / @kingthorin I have updated when the panel and listener are initialized, could you please check if this is a good solution?

**Comment by thc202** [comment]
Yes, that looks good to me, thank you!

**Comment by kingthorin** [comment]
Looks good to me. I guess the conflict just needs to be addressed, then hopefully this could be merged?

**Comment by kingthorin** [comment]
@bugcrowd is there something we can do to get this moving?

**Comment by kingthorin** [comment]
@cak I guess they want you to resubmit

**Pull request merged**